test/features/device.feature

```diff
@@ -17,11 +17,11 @@ Scenario: Device data is included by default
   And the event "device.orientation" matches "landscape|portrait"
   And the event "device.time" is not null
   And the event "device.runtimeVersions.reactNative" matches "\d+\.\d+\.\d"
-  And the event "device.runtimeVersions.expoApp" matches "\d+\.\d+\.\d"
+  And the event "device.runtimeVersions.expoApp" is null
```

Comment:
Do we specifically want to verify that expoApp and appOwnership and null, or are they just not applicable any more (in which case I would remove the lines completely)?

Reply:
That's a good question which I had thought but left until review @gingerbenw any input here?

Reply:
I vote for remove ✋🏻